Makefile

```diff
@@ -22,6 +22,9 @@ licenser:
 gomod:
 	go mod tidy
 
+readme:
```

Comment:
Well, it's generating the README, not checking anything, so I thought it should be it's own target. 

I was also thinking we could have a more general `update` target that depended on this `readme` target OR an `update` target that directly called the README generation script. WDYT?

Reply:
It's rather a question about preserving consistency: is it possible to add new command and forget to regenerate README (something similar to what we have in integrations for docs).

Reply:
Oh, I see what you mean. You want `check` to check if the README is up to date and return an error if it's not? I'm fine with that, but we'll also need a separate target to actually re-generate the README. That's what the `readme` target's intent was but maybe we should rename it to `update` to keep it more general (in case we have more such updates in the future)?

Reply:
I'm fine with both names, you can pick the one, you think, fits better :)